tests/app/main/views/service_settings/test_service_settings.py

```diff
@@ -291,7 +292,7 @@ def test_organisation_name_links_to_org_dashboard(
     assert normalize_spaces(org_row.find('a').text) == 'Test organisation'
 
 
-@pytest.mark.parametrize('service_contact_link,expected_text', [
+@ pytest.mark.parametrize('service_contact_link,expected_text', [
```

Comment:
[question] this seems weird, is this space supposed to be here?

Reply:
Super weird, definitely looks like a fat-fingered cmd-f/r, but it didn't change _all_ the instances of `@pytest` in this file... In any case, this is fixed

Reply:
Looks like there some more with `@ freeze_time`

Reply:
Haha, I'm too deep in it now to see the forest for the trees, thanks for that. Should also be fixed.